chunky/src/java/se/llbit/math/BVH.java

```diff
@@ -43,10 +45,10 @@
     SAH_MA,
   }
 
-
-  static abstract class Node {
-    protected final AABB bb;
-    protected final Primitive[] primitives;
+  @PluginApi
```

Comment:
Hm… Yes this will break plugins when changed but I wouldn't consider this to be part of the plugin api :thinking:

Reply:
I replaced these with a javadoc saying that they are public for some plugins but that stability is not guaranteed.